neuro-sdk/src/neuro_sdk/_jobs.py

```diff
@@ -518,6 +519,8 @@ async def list(
             params.add("being_dropped", str(_being_dropped))
         if _logs_removed is not None:
             params.add("logs_removed", str(_logs_removed))
+        for org_name in org_names:
+            params.add("org_name", org_name or "NO_ORG")
```

Comment:
As I could see, we do not support `NO_ORG` filter on the API side yet.

Reply:
submited https://github.com/neuro-inc/platform-api/issues/2096

Reply:
thanks, have already started working on it